src/NServiceBus.AcceptanceTests/Core/Installers/When_installing_endpoint.cs

```diff
@@ -26,11 +25,8 @@ public async Task Should_only_execute_setup_and_complete()
             Assert.That(context.FeatureSetupCalled, Is.True, "Should initialize Features");
             Assert.That(context.FeatureStartupTaskCalled, Is.False, "Should not start FeatureStartupTasks");
         }
-        Assert.That(new string[]
-        {
-            $"{nameof(TransportDefinition)}.{nameof(TransportDefinition.Initialize)}",
-            $"{nameof(IMessageReceiver)}.{nameof(IMessageReceiver.Initialize)} for receiver Main",
-        }, Is.EqualTo(context.TransportStartupSequence).AsCollection, "Should not start the receivers");
+
+        Assert.That(new[] { $"{nameof(TransportDefinition)}.{nameof(TransportDefinition.Initialize)}", $"{nameof(IMessageReceiver)}.{nameof(IMessageReceiver.Initialize)} for receiver Main", }, Is.EqualTo(context.TransportStartupSequence).AsCollection, "Should not start the receivers");
```

Comment:
This seems to be less readable than before. I would vote for undoing this change